tools/android_lint/project.xml

```diff
@@ -135,6 +90,7 @@
     <src file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/systemchannels/TextInputChannel.java" />
     <src file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/systemchannels/AccessibilityChannel.java" />
     <src file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/systemchannels/KeyEventChannel.java" />
+    <src file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/systemchannels/SpellCheckChannel.java" />
```

Comment:
it looks like this file can get out of sync. what about adding some code to ensure that if this file has uncommitted changes in the current git tree then `android_lint` fails?

It could go here: https://github.com/flutter/engine/blob/2962099077b36704071802ea5595f4a016a1c214/tools/android_lint/bin/main.dart

Reply:
this is a separate issue, so if you want to do this, feel free to do it in a different PR